### PR TITLE
Changes beginning text, makes  you start in cryo pods, makes the captain start in his pod, naked, makes me commit suicide because git sucks ass

### PR DESCRIPTION
**Pull request by karmaisblackandbluepilled**
test ingame before merging thanks

### CODE REVIEW

File: code/game/jobs/job_controller.dm [hunk 1]
```diff
@@ -536,16 +536,37 @@ var/global/datum/controller/occupations/job_master
 				W.set_dir(H.dir)
 				W.buckled_mob = H
 				W.add_fingerprint(H)
+		to_chat(H, "<B>.......<B>")
+		spawn(20)
+			to_chat(H, "<B>Wh-where am I?</B>")
+			spawn(20)
+				to_chat(H, "<B>.......<B> ")
+				spawn(20)
+					to_chat(H, "<B>Right, right...I'm [job.total_positions == 1 ? "the" : "a"] [alt_title ? alt_title : rank] working for Tetracorp on one of their ''state of the art'' research outposts. ")
```
Comment (nothing-now) [inline code/game/jobs/job_controller.dm:545]:
```suggestion
					to_chat(H, "<B>Right, right...I'm [job.total_positions == 1 ? "the" : "a"] [alt_title ? alt_title : rank] working for Tetracorp on one of their \"state of the art\" research outposts. ")
```

**Comment by karmaisblackandbluepilled** [comment]
ok done

**Comment by karmaisblackandbluepilled** [comment]
hold on some shit was fucked because i'm retarded

**Comment by SireTurret2** [comment]
did you fix it

**Comment by karmaisblackandbluepilled** [comment]
yeah the issue is that i merged something completely unrelated here and it fucked my ass

**Comment by karmaisblackandbluepilled** [comment]
this is why this pr does 4 more things than before

**Pull request merged**